Lib/openbakery/profiles/fvar.py

```diff
@@ -448,6 +448,57 @@ def com_adobe_fonts_check_varfont_valid_postscript_nameid(ttFont, has_name_table
         yield PASS, "All postScriptNameID values are valid."
 
 
+@check(
+    id="com.adobe.fonts/check/varfont/valid_instance_postscript_name",
+    rationale="""
+        According to the 'name' documentation in the OpenType Spec
+        https://learn.microsoft.com/en-us/typography/opentype/spec/name
+
+        Postscript names may contain only the characters a-z A-Z 0-9 - and _
+        This check validates the postscript names of a VF's instances specifically.
+    """,
+    conditions=["is_variable_font"],
+    proposal="https://github.com/googlefonts/fontbakery/issues/3704",
+)
+def com_adobe_fonts_check_varfont_valid_instance_postscript_name(
+    ttFont, has_name_table
+):
+    """Validates that all instances in a VF have valid PostScript names."""
+    from openbakery.profiles.name import valid_postscript_name
+
+    if not has_name_table:
+        yield FAIL, Message("lacks-table", "Font lacks 'name' table.")
+        return
```

Comment:
nitpick: add a test to provide coverage for these lines. You can probably copy the code from another test